### ISSUE TITLE
IE11 Action Link, hover state Inverse background.

### ISSUE THREAD

**Issue by chris-gds** [alphagov/govuk_publishing_components#2125]
## Problem

While reviewing [this PR](https://github.com/alphagov/frontend/pull/2806) I noticed that the Hover State for Action Links on Inverse Background is missing on IE11.  This is a problem as users on IE11 are offered no indication there is a change resulting in an a11y issue.

https://components.publishing.service.gov.uk/component-guide/action_link/simple_light_arrow/preview

## Solution

Update so these links on IE11 receive a colour change, grey seem to be in use for other white links on dark backgrounds.

## Visuals (IE11)
![Jun-09-2021 13-48-25](https://user-images.githubusercontent.com/71266765/121358419-84dfc900-c92a-11eb-9c29-5f2725e56da7.gif)

![Jun-09-2021 13-52-22](https://user-images.githubusercontent.com/71266765/121358401-827d6f00-c92a-11eb-8e38-f4af2d497dcd.gif)

**Comment by owenatgov** [comment]
Planning on pairing with our new FE on this next week as an intro to the components gem

**Comment by owenatgov** [comment]
I asked about this in the design system channel and got the following discussion:

Me: 
>Hey folks. I was having a look at inverted links and noticed that it doesn't include a colour change on hover. Is this intentional? It feels risky to me as this means that users on browser that don't support changes in line thickness (IE11 and below) won't receive any cue that they're hovering on a link.

Ollie Byford:
>@christopher.thomas any thoughts? We discussed colour changes in https://github.com/alphagov/govuk-design-system/issues/1578 but only really focused on whether or not to keep the existing colour change for ‘standard links’. I’m sure we talked about inverse links, and it was a conscious choice to leave them without a colour change, but can’t remember the rationale…

Chris Thomas
>From memory I feel like there was a problem with using opacity to lighten them as opposed to a specified colour? So we couldn’t make an inverse link hover colour that would work across all kinds of different backgrounds? Not 100% on this though, I might have to do more digging. I think on the IE11 thing we were satisfied that the pointer cursor was enough of an indicator even if not as clear as the proper hover state

Me
>I seeeeeee. Cheers both. Being frank, I have opinions about the inconsistency this creates but I ain't no design systemer. 
Couple of follow ups:
Do we have any data on if user's are ok with the cursor as the sole indicator?
How do you feel about a given team adding a custom colour change for their product/service?

Max Harden
>2 feels like a bad idea to me, Owen. Fragments things, more stuff to get forgotten/in the way of updates, brittle if the underlying code changes.

Ollie Byford
>So, it’s pretty hard to get any hard data around this kind of thing. Even in user research, all you can really evaluate is whether the users you’re doing research with manage to use the thing OK. I would note that the ‘text-colored’ links (like breadcrumbs, back links) have never had any sort of hover state before, and inverse links generally haven’t either. So we have definitely been seeing the text underline change as an enhancement. Also worth noting that there are plenty of other interactive elements that have no hover state at all (like checkboxes and radios) where the only change is the cursor.

**Comment by chris-gds** [comment]
> I’m sure we talked about inverse links, and it was a conscious choice to leave them without a colour change, but can’t remember the rationale…

> From memory I feel like there was a problem with using opacity to lighten them as opposed to a specified colour? So we couldn’t make an inverse link hover colour that would work across all kinds of different backgrounds? Not 100% on this though, I might have to do more digging. 

> I think on the IE11 thing we were satisfied that the pointer cursor was enough of an indicator even if not as clear as the proper hover state 

At this stage, I still think this is worth resolving to align consistency and as IE11 is still on the service manual.

**Comment by owenatgov** [comment]
Revisiting this, this issue may become redundant quite soon as the new homepage design Explore are working on will be removing this section outright. See https://github.com/alphagov/frontend/pull/3028

**Comment by chris-gds** [comment]
Homepage redesign is live - closing issue
